src/main/drivers/accgyro/accgyro.h

```diff
@@ -58,6 +58,9 @@ typedef struct gyroDev_s {
     int32_t gyroZero[XYZ_AXIS_COUNT];
     int32_t gyroADC[XYZ_AXIS_COUNT];                        // gyro data after calibration and alignment
     int16_t gyroADCRaw[XYZ_AXIS_COUNT];
+#if defined(USE_GYRO_SLEW_LIMITER)
```

Comment:
This will break, since `USE_GYRO_SLEW_LIMITER` is never defined at the point that this header file is included.

Reply:
Yes, just include `gyroADCRawPrevious` unconditionally here. The cost is minimal.